src/index.js

```diff
@@ -63,6 +64,10 @@ function IpfsAPI (host_or_multiaddr, port) {
       opts = {}
     }
 
+    if (typeof files === 'string' && files.startsWith('http')) {
+      files = request(files)
```

Comment:
Not sure what your alternative scenario would look like, we can't stream directly to go ipfs anyway